packages/@mollitia/redis/README.md

```diff
@@ -0,0 +1,58 @@
+# Mollitia
+
+<p align="center"><br/><img width="200" src="https://genesys.github.io/mollitia/favicon.svg" alt="Mollitia Icon"/><br/><br/></p>
+
+> Mollitia - Redis Addon
+
+The `Mollitia` [Redis](https://redis.io/) addon adds redis for some modules of every circuit. The list of modules coming with redis support are Ratelimit, SlidingCountBreaker and SlidingTimeBreaker.
+
+## 📄 Documentation
+
+Please check out the official documentation to get started using **Mollitia**, visit [genesys.github.io/mollitia](https://genesys.github.io/mollitia).
+
+## ⚙️ Installation
+
+``` bash
+npm install --save @mollitia/redis
+```
+
+## 🚀 Usage
+
+``` typescript
+// Imports the library
+import * as Mollitia from 'mollitia';
+import { RedisAddOn } from '@mollitia/redis';
```

Comment:
To keep it coherent will `Mollitia.Addon` and `PrometheusAddon`, we should probably go for exporting `RedisAddon` here.

There's a few occurrences in readme or ts files, I'll leave this as one comment though.